content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md

```diff
@@ -0,0 +1,147 @@
+
+![07-volume-manage](/images/blogs/en/release-announcement3.2.0/07-volume-manage.png)
+
+The default immediate binding mode binds a volume to a backend storage device immediately when the volume is created. This mode does not apply to storage devices with topology limits and may cause Pod scheduling failures. KubeSphere 3.2.0 provides the delayed binding mode to address this issue, which guarantees that a volume (PVC) is bound to a volume instance (PV) only after the volume is mounted to a Pod. This feature ensures that resources are properly scheduled based on Pod resource requests.
+
+![08-storage-class-settings](/images/blogs/en/release-announcement3.2.0/08-storage-class-settings.png)
+
+In addition to volume management, KubeSphere 3.2.0 now also supports volume instance management, and you can view volume instance information, edit volume instances, and delete volume instances on the web console.
+
+![09-volumes](/images/blogs/en/release-announcement3.2.0/09-volumes.png)
+
+When you create a volume snapshot, you can specify the snapshot type (`VolumeSnapshotClass`) to use a specific storage backend.
+
+### **Cluster gateway**
+
+KubeSphere 3.1 supports only project gateways, which require multiple IP addresses when there are multiple projects. Additionally, gateways in different workspaces are independent.
+
+KubeSphere 3.2.0 provides a cluster gateway, which means that all projects can share the same gateway. Existing project gateways are not affected by the cluster gateway.
+
+![10-gateway-settings](/images/blogs/en/release-announcement3.2.0/10-gateway-settings.png)
+
+The administrator can directly manage and configure all project gateways on the cluster gateway settings page without having to go to each workspace. The Kubernetes ecosystem provides many ingress controllers that can be used as the gateway solution. In KubeSphere 3.2.0, the gateway backend is refactored, which allows you to use any ingress controllers that support v1\\ingress as the gateway solution.
+
+![11-gateway-settings2](/images/blogs/en/release-announcement3.2.0/11-gateway-settings2.png)
+
+### **Authentication and authorization**
+
+A unified and all-round identity management and authentication system is indispensable for logical isolation in a multi-tenant system. Apart from support for AD/LDAP and OAuth 2.0 identity authentication systems, KubeSphere 3.2.0 also provides a built-in authentication service based on OpenID Connect to provide authentication capability for other components. OpenID Connect is a simple user identity authentication protocol based on OAuth 2.0 with a bunch of features and security options to meet enterprise-grade business requirements.
+
+### **App Store open to community partners**
```

Comment:
IMO, if we use `open` as a verb, we might use `opens`

Reply:
3KS for your advice. But here "open" is used as an adjective

Reply:
Will `App Store is open to community partners` looks much better?

Reply:
I think 'is' can be omitted here

Reply:
Okay.